config/service_alias.go

```diff
@@ -30,63 +30,82 @@ func init() {
 	for k, v := range legacyMorseQoSTypes {
 		ServiceQoSTypes[k] = v
 	}
+	for k, v := range testQoSTypes {
+		ServiceQoSTypes[k] = v
+	}
 }
 
+// Shannon service IDs.
+// As of 12/2024, these service IDs are on Beta TestNet and intended to be moved
+// over to MainNet once the network is ready.
 var shannonQoSTypes = map[protocol.ServiceID]ServiceQoSType{
-	// TODO_IMPROVE Add all non-EVM Morse Services and requisite initialization
-
-	// Shannon Service IDs
-	"anvil": ServiceIDEVM, // ETH Local (development/testing)
-
-	// TODO_IMPROVE(@commoddity): Use actual service IDs for Solana and POKT.
+	// Solana Service IDs
 	"solana": ServiceIDSolana,
 
+	// EVM Service IDs
+	"eth": ServiceIDEVM,
+
+	// POKT Service IDs
 	"pokt":  ServiceIDPOKT,
 	"morse": ServiceIDPOKT,
+}
+
+// Shannon test service IDs.
+// As of 12/2024, these service IDs are on Beta TestNet and primarily used
+// for E2E testing. They may or may not be moved over to MainNet once the network.
+var testQoSTypes = map[protocol.ServiceID]ServiceQoSType{
+	// Shannon Service IDs
+	"anvil": ServiceIDEVM, // ETH Local (development/testing)
 
 	// Gateway E2E service ID is used only for running PATH's Morse and Shannon E2E tests.
```

Comment:
Since we are cleaning up this file, I suggested to remove `gatewaye2e` entry as it is not used anymore.